lms/djangoapps/grades/management/commands/reset_grades.py

```diff
@@ -89,8 +95,16 @@ def handle(self, *args, **options):
 
         operation = self._query_grades if run_mode == 'dry_run' else self._delete_grades
 
-        operation(PersistentSubsectionGrade, course_keys, modified_start, modified_end)
-        operation(PersistentCourseGrade, course_keys, modified_start, modified_end)
+        db_table = options.get('db_table')
+
+        if db_table not in {'subsection', 'course', None}:
```

Comment:
Nit: Can you move this validation to L82 (on the top of the function)?  As all options validation is done before the rest of the business logic.